ui/components/multichain/account-list-menu/account-list-menu.js

```diff
@@ -73,7 +73,7 @@ const ACTION_MODES = {
  * @param internalAccounts - internal accounts
  * @returns merged accounts list with balances and internal account data
  */
-const mergeAccounts = (accountsWithBalances, internalAccounts) => {
+export const mergeAccounts = (accountsWithBalances, internalAccounts) => {
```

Comment:
@k-g-j since we are using this logic in multiple files now, is there a better place we can put it? perhaps now would be a good time to implement a new selector?

Reply:
Out of scope for this PR, but I think this should be a selector instead of a utility function for sure!

Reply:
We wont need this function anymore once #21709 is merged so we will need to remember to go back and refactor where we are using this function to remove it because the new way identities are formed will perform this action.